### DISCUSSION TITLE
Auto highlight *Origin* and *Size* text on selection

### DISCUSSION THREAD

**Discussion by CaptainCooky** [Mojang/minecraft-editor#61] [Feedback]
This would be a nice little quality-of-life feature. 

When selecting the values in the text fields of both the *Origin* and *Size*, it would be great if the pre-existing values were auto-highlighted. 
This would be helpful when making complex selections. 

e.g. When using the `ctrl` modifier to add to a selection, we could highlight the first block in a section, then click on an axis value and change the size value without selecting or deleting the existing text. A small thing, but it would greatly improve workflow.

**Comment by minecraftchloe** [comment]
Fantastic idea!
